plugins/pipelines/listEvents.js

```diff
@@ -23,14 +23,20 @@ module.exports = () => ({
         },
         handler: (request, reply) => {
             const factory = request.server.app.pipelineFactory;
-
+            
```

Comment:
Unsure but is this extra whitespace?

Reply:
lint requires blank line after variable declaration

Reply:
I think Filbert is referring to the extra spaces in that line